_posts/newsletter/2021-09-09-this-month-in-solid.md

```diff
@@ -0,0 +1,42 @@
+
+Our final speaker was Frank Smutniak, who showed us how Solid could help you create a healthy garden. With the help of different materials and using Solid Pods, we learned that you could measure moisture levels, report additional measurements and the pump’s status. All of this helps you to adjust how you plant and care for your garden. One of his next steps includes installing and hosting the application on the Community Solid Server. In the coming weeks, he plans to input the information from the presentation to a [Github repo](https://github.com/itgrows/iot_garden.git), so keep an eye out for that. 
+
+It’s amazing all that’s capable with Solid, especially when building applications. After hearing about these two unique apps and the positive feedback next month, we decided to spin off the theme to include additional apps that are using Solid. Our next Solid World will be on October 7th! To continue this conversation, join us on the [Solid Forum](https://forum.solidproject.org/) or the [Solid World Gitter](https://gitter.im/solid/SolidWorld). Want to participate in these conversations and present at a future Solid World? Fill out the [Solid World speaker form](https://es1cz4pb7oi.typeform.com/to/nietD34f). Lastly, the Solid Team can address any questions, comments, or concerns at info@solidproject.org. We hope to see you next month!
+
+
+### In Other News
+
+* Solid Networking Event – Every last Friday of the month. The upcoming gathering is on September 25, 2021 at 11 AM Eastern Time
+
+#### Community Spotlight
+
+* Vincent Tunru who is a benefit to the community for staying involved!
+* AJamesPhillips (username) who is very engaged in the community
+
+### Opportunities
+* Recruitment Available: We are strengthening our team - www.instans.net - we are bidding for a contract with the Scottish Government to build a MVP for a carbon emissions tracking app, for companies to manage CO2 emissions in their supply chain.
+
+We would like to know who is out there, and who might be interested in participating with us in a 4 month accelerator Oct 2021 to Jan 2022. Ideally, a full stack developer experienced with Node.js, SOLID, handling login and authentication. Experience (or interest in learning) search and query would be beneficial.
+
+We are an equal opportunities employer and value diversity. contact@instans.net
```

Comment:
Looking at https://deploy-preview-627--solidproject-dot-org.netlify.app/newsletter/2021-09-09, this doesn't all line up with the bullet:

![image](https://user-images.githubusercontent.com/4251/132720449-f1f3f1ac-9565-4a6d-be87-75271f00a62b.png)

I'm not sure if this solves it, but [it might](https://stackoverflow.com/questions/33673639/multi-line-bullet-list-in-markdown). Also, adding `https://` to turn it into a clickable link, and corrected the spelling of Solid:

```suggestion
* Recruitment Available: We are strengthening our team - https://www.instans.net - we are bidding for a contract with the Scottish Government to build a MVP for a carbon emissions tracking app, for companies to manage CO2 emissions in their supply chain.\
We would like to know who is out there, and who might be interested in participating with us in a 4 month accelerator Oct 2021 to Jan 2022. Ideally, a full stack developer experienced with Node.js, Solid, handling login and authentication. Experience (or interest in learning) search and query would be beneficial.\
We are an equal opportunities employer and value diversity. contact@instans.net
```